### ISSUE TITLE
Add support for tags

### ISSUE THREAD

**Issue by jimmybot** [honeybadger-io/honeybadger-js#554]
Other Honeybadger libraries support sending tags as part of the context.  In order to be as useful as possible, the JS library should do the same.

Examples:
Ruby: https://docs.honeybadger.io/lib/ruby/getting-started/tagging-errors.html
Elixir: https://github.com/honeybadger-io/honeybadger-elixir/blob/3c8656a59d05d5e39bcf53e5fe543a5e468f6707/lib/honeybadger/notice.ex

**Comment by joshuap** [comment]
I'd like to support both of these APIs:

```
// Add tags to all errors via context (a shorthand that we support in the Ruby client):
Honeybadger.context({
   tags: 'foo, bar'
})

// Add tags to a specific error:
Honeybadger.notify(error, {
  tags: 'foo, bar'
})
```

Also, let's support both tags as a comma-delimited string (`"foo, bar"`), and an array (`["foo", "bar"]`).

**Comment by joshuap** [comment]
Once this is done, we should also add a guide to the JavaScript docs similar to this Ruby guide:

https://docs.honeybadger.io/lib/ruby/getting-started/tagging-errors.html

and add `notice.tags` to the list of notice properties:

https://docs.honeybadger.io/lib/javascript/guides/customizing-error-reports.html#modifying-data

**Comment by joshuap** [comment]
@shipjs prepare

**Comment by joshuap** [comment]
Released in 3.2.0

Docs: http://localhost:4000/lib/javascript/guides/tagging-errors.html
